client/widgets/db-stats/stats.template.html.slim

```diff
@@ -31,26 +29,32 @@
             tr
               th.rows.text-right Count
               th.rows-trend.text-center.db-stats-right-border Trend
-              th.total-size.text-right Total
+              th.total-size.text-right Size
           tbody
-            tr ng-repeat='stat in stats'
+            tr ng-repeat='stat in currentStats'
               td.name.db-stats-right-border {{ stat.name }}
```

Comment:
Some table names are quite long. Investigate these solutions:
* Use CSS to allow word breaks: `word-break: break-word`.
* Replace underscores with spaces in `xs` size (or in all sizes).
* Smaller font for table names.